tests/debug-mode/host/host.c

```diff
@@ -68,10 +68,11 @@ static void _test_debug_signed(const char* path)
 {
     /* Signed debug mode should always pass. */
     _launch_enclave_success(path, _create_flags(true));
-#ifdef OE_HAS_SGX_DCAP_QL
-    /* Only works with FLC */
-    _launch_enclave_success(path, _create_flags(false));
-#endif
+    if (oe_has_sgx_quote_provider())
```

Comment:
This is an unfortunate example of code semantics having shifted out from under this original implementation. This was originally intended as a compile time check for whether the project was compiled for SGX1 or SGX1FLC configurations, but the flag has since been repurposed after the loader stack stopped making that distinction.

What this test cares about is that the launch control stack, and whether the quote provider is installed or not is irrelevant. For CI/CD purposes, it just so happens they coincide, so this is a proxy signal for the whether it should run the production signed launch test or not. 

We should file a separate issue to have this fixed up as a CI/CD task:
- [ ] This test should take an explicit environment variable for `SKIP_NON_DEBUG_LAUNCH` and choose to execute these tests or not
- [ ] CI/CD infrastructure should set this environment variable for the SGX1 configuration pipelines.

Reply:
Good catch!
I have created issue #3287